<?xml version="1.0" encoding="UTF-8"?>
<!--
  - Copyright (c) 2022 船山信息 chuanshaninfo.com
  - The project is licensed under Mulan PubL v2.
  - You can use this software according to the terms and conditions of the Mulan
  - PubL v2. You may obtain a copy of Mulan PubL v2 at:
  -          http://license.coscl.org.cn/MulanPubL-2.0
  - THIS SOFTWARE IS PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND,
  - EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO NON-INFRINGEMENT,
  - MERCHANTABILITY OR FIT FOR A PARTICULAR PURPOSE.
  - See the Mulan PubL v2 for more details.
  -->

<ui version="4.0">
 <class>GeneralForm</class>
 <widget class="QWidget" name="GeneralForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
          <number>0</number>
      </property>
   <property name="leftMargin">
       <number>0</number>
   </property>
   <property name="topMargin">
       <number>0</number>
   </property>
   <property name="rightMargin">
       <number>0</number>
   </property>
   <property name="bottomMargin">
       <number>0</number>
   </property>
   <item>
       <widget class="QGroupBox" name="generalGroup">
           <property name="title">
               <string/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                   <layout class="QHBoxLayout" name="generalLayout">
                       <item>
                           <widget class="QLabel" name="transLabel">
                               <property name="toolTip">
                                   <string>The translation may not load until qTox restarts.</string>
                               </property>
                               <property name="text">
                                   <string>Language:</string>
                               </property>
                           </widget>
                       </item>
                       <item>
                           <widget class="QComboBox" name="transComboBox">
                               <property name="sizePolicy">
                                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                                       <horstretch>0</horstretch>
                                       <verstretch>0</verstretch>
                                   </sizepolicy>
                               </property>
                               <property name="toolTip">
                                   <string>The translation may not load until qTox restarts.</string>
                               </property>
                           </widget>
                       </item>
                       <item>
                           <spacer name="generalSpacer">
                               <property name="orientation">
                                   <enum>Qt::Horizontal</enum>
                               </property>
                               <property name="sizeHint" stdset="0">
                                   <size>
                                       <width>40</width>
                                       <height>20</height>
                                   </size>
                               </property>
                           </spacer>
                       </item>
                   </layout>
               </item>
               <item>
                   <layout class="QVBoxLayout" name="startupLayout">
                       <item>
                           <widget class="QCheckBox" name="cbAutorun">
                               <property name="toolTip">
                                   <string>Start qTox on operating system startup (current profile).</string>
                               </property>
                               <property name="text">
                                   <string>Autostart</string>
                               </property>
                           </widget>
                       </item>
                       <item>
                           <widget class="QCheckBox" name="checkUpdates">
                               <property name="text">
                                   <string>Check for updates</string>
                               </property>
                           </widget>
                       </item>
                   </layout>
               </item>
               <item>
                   <layout class="QVBoxLayout" name="verticalLayout_6">
                       <item>
                           <widget class="QCheckBox" name="cbSpellChecking">
                               <property name="text">
                                   <string>Spell checking</string>
                               </property>
                           </widget>
                       </item>
                       <item>
                           <widget class="QCheckBox" name="showSystemTray">
                               <property name="enabled">
                                   <bool>true</bool>
                               </property>
                               <property name="sizePolicy">
                                   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                                       <horstretch>0</horstretch>
                                       <verstretch>0</verstretch>
                                   </sizepolicy>
                               </property>
                               <property name="text">
                                   <string>Show system tray icon</string>
                               </property>
                           </widget>
                       </item>
                       <item>
                           <layout class="QVBoxLayout" name="verticalLayout_8">
                               <property name="leftMargin">
                                   <number>40</number>
                               </property>
                               <item>
                                   <widget class="QCheckBox" name="startInTray">
                                       <property name="sizePolicy">
                                           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                                               <horstretch>0</horstretch>
                                               <verstretch>0</verstretch>
                                           </sizepolicy>
                                       </property>
                                       <property name="toolTip">
                                           <string comment="toolTip for Start in tray setting">qTox will start minimized
                                               in tray.
                                           </string>
                                       </property>
                                       <property name="text">
                                           <string>Start in tray</string>
                                       </property>
                                   </widget>
                               </item>
                               <item>
                                   <widget class="QCheckBox" name="minimizeToTray">
                                       <property name="sizePolicy">
                                           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                                               <horstretch>0</horstretch>
                                               <verstretch>0</verstretch>
                                           </sizepolicy>
                                       </property>
                                       <property name="toolTip">
                                           <string comment="toolTip for minimize to tray setting">After pressing
                                               minimize (_) qTox will minimize itself to tray,
instead of system taskbar.</string>
                                       </property>
                                       <property name="text">
                                           <string>Minimize to tray</string>
                                       </property>
                                   </widget>
                               </item>
                               <item>
                                   <widget class="QCheckBox" name="closeToTray">
                                       <property name="sizePolicy">
                                           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                                               <horstretch>0</horstretch>
                                               <verstretch>0</verstretch>
                                           </sizepolicy>
                                       </property>
                                       <property name="toolTip">
                                           <string comment="toolTip for close to tray setting">After pressing close (X)
                                               qTox will minimize to tray,
instead of closing itself.</string>
                                       </property>
                                       <property name="text">
                                           <string>Close to tray</string>
                                       </property>
                                   </widget>
                               </item>
                           </layout>
                       </item>
                   </layout>
               </item>
               <item>
                   <spacer name="verticalSpacer">
                       <property name="orientation">
                           <enum>Qt::Vertical</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                           <size>
                               <width>20</width>
                               <height>40</height>
                           </size>
                       </property>
                   </spacer>
               </item>
           </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>transComboBox</tabstop>
  <tabstop>cbAutorun</tabstop>
  <tabstop>checkUpdates</tabstop>
  <tabstop>showSystemTray</tabstop>
  <tabstop>startInTray</tabstop>
  <tabstop>minimizeToTray</tabstop>
  <tabstop>closeToTray</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>showSystemTray</sender>
   <signal>toggled(bool)</signal>
   <receiver>closeToTray</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showSystemTray</sender>
   <signal>toggled(bool)</signal>
   <receiver>minimizeToTray</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showSystemTray</sender>
   <signal>toggled(bool)</signal>
   <receiver>startInTray</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
